### PR TITLE
Update TD Ameritrade TFA status 

### PR DESCRIPTION
**Pull request by ItsMattL**
Instructions require account login...
https://invest.ameritrade.com/cgi-bin/apps/u/help/security/protectingYourAccountAndPersonalData/twoFactorAuthentication

### CODE REVIEW

File: _data/investing.yml [hunk 1]
```diff
@@ -174,7 +174,8 @@ websites:
       url: https://www.tdameritrade.com
       twitter: TDAmeritrade
```
Comment (mxxcon) [inline _data/investing.yml]: Please remove twitter line since it's no longer needed.

File: _data/investing.yml [hunk 2]
```diff
@@ -174,7 +174,8 @@ websites:
...
       img: tdameritrade.png
-      tfa: No
+      tfa: Yes
+      sms: Yes
```
Comment (mxxcon) [inline _data/investing.yml:178]: Please add your url or another publicly accessible one as `doc:` line

**Review by mxxcon — CHANGES_REQUESTED** [review]

**Review by mxxcon — DISMISSED** [review]

**Review by jamcat22 — DISMISSED** [review]
Because the doc link requires logging in, should we add an exception to state this, or create a note to replace it? Could someone with a TD Ameritrade account post a screenshot of the doc link (blurring out any sensitive info)?

**Comment by ItsMattL** [comment]
Sure, here you go.

ameritrade help

**Comment by mxxcon** [comment]
@jamcat22 i don't think we've done that before...We generally prefer publicly accessible URls, but if not it's ok too..

**Comment by ItsMattL** [comment]
Anything else I can do here?

**Comment by Carlgo11** [comment]
@mxxcon I disagree, I've seen many PRs where we have simply left out the doc page because it's behind a login screen. In fact I think most sites listed here that only list documentation for registered users are without a doc link.

**Comment by ItsMattL** [comment]
Ping... 

I'm amenable to however the project owners want to manage the documentation, but I think it's unfortunate that in the mean time, the site has been providing outdated information for multiple months now. :\

**Comment by brycx** [comment]
Hi @ItsMattL 
It is unfortunate, and we all try our best with the amount of time we can put aside for reviewing. That's no excuse of course for PR's to be left behind. I appreciate the ping.

I agree with @Carlgo11, most sites with walled doc-links aren't given the `doc:` attribute. Please remove the doc link.

**Review by brycx — APPROVED** [review]

**Review by stephengroat — APPROVED** [review]

**Pull request merged**